### PR TITLE
Update pipeline to include smoke & e2e test

### PR DESCRIPTION
**Pull request by jameshill**
I noticed that I wasn't running all the tests in the suite... only the unit & acceptance tests.

I've updated the pipeline to run an everything else step which will run everything that's not a unit or acceptance test. 

Hopefully testing the exclude pattern.

### CODE REVIEW

**Comment by jameshill** [comment]
Hey @buildkite/test-splitting is this the correct usage of the EXCLUDE flag?

**Comment by wooly** [comment]
It looks reasonable, yep, but I can't say for sure that it's going to work. I checked the bk detail for the build and it doesn't look like the build is using the test splitter, is that expected?

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -31,6 +31,15 @@ steps:
...
       BUILDKITE_SPLITTER_TEST_FILE_PATTERN: "spec/acceptance/**/*_spec.rb"
 
+  - label: "Everything Else"
+    command: .buildkite/steps/run
+    key: everything_else
+    agents:
+      queue: "hosted-queue"
+    parallelism: 2
+    env:
+      BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN: "{spec/acceptance/**/*_spec.rb,spec/unit/**/*_spec.rb}"
```
Comment (nprizal) [inline .buildkite/pipeline.yml]: alternatively, you can do this:
```suggestion
      BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN: "spec/{acceptance,unit}/**/*_spec.rb"
```

**Review by nprizal — APPROVED** [review]
Can we create build from branch to see if it works first?

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (jameshill) [reply .buildkite/pipeline.yml]: oh yeah, that's much nicer

**Comment by jameshill** [comment]
Ran a build of this [branch](https://buildkite.com/test-analytics-sandbox/example-test-suite-ruby/builds/475#_) and it looks like it didn't run it.

**Comment by jameshill** [comment]
> Ran a build of this [branch](https://buildkite.com/test-analytics-sandbox/example-test-suite-ruby/builds/475#_) and it looks like it didn't run it.

Of course, I didn't pass it through to docker 🤣 updated now and all good.

**Pull request merged**